pytest_plugins/fixture_markers.py

```diff
@@ -42,19 +42,52 @@ def pytest_generate_tests(metafunc):
                     if re.fullmatch(str(matcher[0]), str(setting_rhel_ver))
                 ]
             )
+        network_params = []
         rhel_params = []
+        ids = []
         filtered_versions = set(list_params + match_params)
         # default to all supported versions if no filters were found
         for ver in filtered_versions or settings.supportability.content_hosts.rhel.versions:
             rhel_params.append(dict(rhel_version=ver, no_containers=no_containers))
+
         if rhel_params:
             rhel_params.sort(key=lambda r: str(r['rhel_version']))
+            ids = [f'rhel{r["rhel_version"]}' for r in rhel_params]
+        if not no_containers and rhel_params:
+            network_marker = metafunc.definition.get_closest_marker("network")
+            if network_marker and not settings.server.is_ipv6:
+                network_params = network_marker.args[0] if network_marker else []
+            else:
+                network_params.append('ipv4')
```

Comment:
If no network type params then, the network type should be same as satellite network type.

Meaning if ipv4 sat then ipv4 client, same for ipv6.

Reply:
updated !